api/swagger/swagger.yaml

```diff
@@ -529,6 +559,55 @@ definitions:
         type: string
         format: date-time
 
+  TrialPublication:
+    required:
+      - attributes
+    properties:
+      attributes:
+        $ref: '#/definitions/Publication'
+
+  Publication:
+    required:
+      - id
+      - source
+      - source_url
+      - title
+      - abstract
+    properties:
+      id:
+        type: string
+      source:
+        $ref: '#/definitions/Source'
+      created_at:
+        type: string
+        format: date-time
+      updated_at:
+        type: string
+        format: date-time
+      source_url:
+        type: string
+      title:
+        type: string
+      abstract:
+        type: string
+      authors:
+        type: array
+        items: {
+          "type": "string"
+        }
```

Comment:
Fix the styling, instead of:

```
items: {
  "type": "string"
}
```

Use:

```
items:
  type: string
```

There're other lines with this same issue, for example in line 607~609. Please fix them all.